### PR TITLE
Convert template syntax to Twig

### PR DESCRIPTION
**Pull request by VSEphpbb**

### CODE REVIEW

**Comment by battye** [comment]
This weekend I'll pull your branch and test it.

File: styles/prosilver/template/contributions/demo.html [hunk 1]
```diff
@@ -1,97 +1,91 @@
...
 						<div>
-							<!-- IF PREVIEW --><img src="{PREVIEW}" alt="{NAME}" /><!-- ENDIF -->
+							{% if PREVIEW %}<img src="{{ PREVIEW }}" alt="{{ NAME }}" />{% endif %}
 							<ul class="style-details">
-								<li>{L_BY} {AUTHORS}</li>
-								<li>{L_PHPBB_VERSION}: {PHPBB_VERSION}</li>
-								<li>{L_LICENSE}: {LICENSE}</li>
+								<li>{{ lang('BY') }} {{ AUTHORS }}</li>
+								<li>{{ lang('PHPBB_VERSION') ~ lang('COLON') ~ PHPBB_VERSION }}</li>
+								<li>{{ lang('LICENSE') ~ lang('COLON') ~ LICENSE }}</li>
```
Comment (kasimi) [inline styles/prosilver/template/contributions/demo.html]: As @battye mentioned, missing space after colon on these two lines.

File: styles/prosilver/template/contributions/demo.html [hunk 2]
```diff
@@ -1,97 +1,91 @@
...
-			<p class="demo-buttons"><a href="{U_VIEW}" class="button details"><span></span>{L_DETAILS}</a><a href="{U_DOWNLOAD}" class="button download"><span></span>{L_DOWNLOAD}</a></p> 
+			<p class="style-name">{{ NAME }}</p>
+			<p class="demo-buttons"><a href="{{ U_VIEW }}" class="button details"><span></span>{{ lang('DETAILS') }}</a><a href="{{ U_DOWNLOAD }}" class="button download"><span></span>{{ lang('DOWNLOAD') }}</a></p> 
 			<ul class="style-details">
-				<li>{L_BY} {AUTHORS}</li>
-				<li>{L_PHPBB_VERSION}: {PHPBB_VERSION}</li>
-				<li>{L_LICENSE}: {LICENSE}</li>
+				<li>{{ lang('BY') }} {{ AUTHORS }}</li>
+				<li>{{ lang('PHPBB_VERSION') ~ lang('COLON') ~ PHPBB_VERSION }}</li>
+				<li>{{ lang('LICENSE') ~ lang('COLON') ~ LICENSE }}</li>
```
Comment (kasimi) [inline styles/prosilver/template/contributions/demo.html]: As @battye mentioned, missing space after colon on these two lines.

File: styles/prosilver/template/contributions/demo.html [hunk 3]
```diff
@@ -1,97 +1,91 @@
...
+{% for stylerow in loops.stylerow %}
+		<li class="style{{ stylerow.ID }}">
+			<div class="preview">{% if stylerow.PREVIEW %}<img src="" alt="{{ stylerow.NAME }}" data-source="{{ stylerow.PREVIEW }}" />{% endif %}
 				<ul class="style-details">
-					<li>{L_BY} {stylerow.AUTHORS}</li>
-					<li>{L_PHPBB_VERSION}: {stylerow.PHPBB_VERSION}</li>
-					<li>{L_LICENSE}: {stylerow.LICENSE}</li>
+					<li>{{ lang('BY') }} {{ stylerow.AUTHORS }}</li>
+					<li>{{ lang('PHPBB_VERSION') ~ lang('COLON') ~ stylerow.PHPBB_VERSION }}</li>
+					<li>{{ lang('LICENSE') ~ lang('COLON') ~ stylerow.LICENSE }}</li>
```
Comment (kasimi) [inline styles/prosilver/template/contributions/demo.html]: As @battye mentioned, missing space after colon on these two lines.

File: styles/prosilver/template/manage/attention.html [hunk 4]
```diff
@@ -1,66 +1,66 @@
...
 <fieldset class="display-options">
-	<!-- IF S_SELECT_SORT_DAYS -->
-		<label>{L_DISPLAY_POSTS}: {S_SELECT_SORT_DAYS}</label>
-		<label>{L_SORT_BY} {S_SELECT_SORT_KEY}</label><label>{S_SELECT_SORT_DIR}</label>
-	<!-- ENDIF -->
-	<input type="checkbox" name="open" id="open"<!-- IF S_OPEN_CHECKED --> checked="checked"<!-- ENDIF --> value="1" /> <label for="open">{L_OPEN_ITEMS}</label> &nbsp;
-	<input type="checkbox" name="closed" id="closed"<!-- IF S_CLOSED_CHECKED --> checked="checked"<!-- ENDIF --> value="1" /> <label for="closed">{L_CLOSED_ITEMS}</label>
-	<input type="submit" name="sort" value="{L_GO}" class="button2" />
+	{% if S_SELECT_SORT_DAYS %}
+		<label>{{ lang('DISPLAY_POSTS') ~ lang('COLON') ~ S_SELECT_SORT_DAYS }}</label>
```
Comment (kasimi) [inline styles/prosilver/template/manage/attention.html]: As @battye mentioned, missing space after colon

File: styles/prosilver/template/manage/attention_details.html [hunk 5]
```diff
@@ -1,101 +1,101 @@
...
-	<h3 class="clear-top">{L_OPEN_ITEMS}</h3>
-	<!-- BEGIN attention -->
-	<div class="attention<!-- IF attention.S_LAST_ROW --> last<!-- ENDIF -->">
-		<h4>{L_REASON}: {attention.ATTENTION_REASON}</h4>
-		<p class="author">{L_POST_BY_AUTHOR} {attention.REPORTER_USER_FULL} &raquo; {attention.ATTENTION_TIME}</p>
+{% if loops.attention|length %}
+	<h3 class="clear-top">{{ lang('OPEN_ITEMS') }}</h3>
+	{% for attention in loops.attention %}
+	<div class="attention{% if attention.S_LAST_ROW %} last{% endif %}">
+		<h4>{{ lang('REASON') ~ lang('COLON') ~ attention.ATTENTION_REASON }}</h4>
```
Comment (kasimi) [inline styles/prosilver/template/manage/attention_details.html]: As @battye mentioned, missing space after colon

File: styles/prosilver/template/manage/attention_details.html [hunk 6]
```diff
@@ -1,101 +1,101 @@
...
+	<h3 class="clear-top">{{ lang('CLOSED_ITEMS') }}</h3>
 
-	<!-- BEGIN attention_closed -->
-	<div class="attention<!-- IF attention_closed.S_LAST_ROW --> last<!-- ENDIF -->">
-		<h4>{L_REASON}: {attention_closed.ATTENTION_REASON}</h4>
-		<p class="author">{L_POST_BY_AUTHOR} {attention_closed.REPORTER_USER_FULL} &raquo; {attention_closed.ATTENTION_TIME}</p>
-		<p class="author attention-closed">{attention_closed.CLOSED_BY_LABEL} {attention_closed.CLOSER_USER_FULL} &raquo; {attention_closed.ATTENTION_CLOSE_TIME}</p>
+	{% for attention_closed in loops.attention_closed %}
+	<div class="attention{% if attention_closed.S_LAST_ROW %} last{% endif %}">
+		<h4>{{ lang('REASON') ~ lang('COLON') ~ attention_closed.ATTENTION_REASON }}</h4>
```
Comment (kasimi) [inline styles/prosilver/template/manage/attention_details.html]: As @battye mentioned, missing space after colon

File: styles/prosilver/template/manage/queue_validate.html [hunk 7]
```diff
@@ -1,66 +1,66 @@
...
-<!-- IF S_DISPLAY_PREVIEW --><!-- INCLUDE posting/posting_preview.html --><!-- ENDIF -->
+{% if S_DISPLAY_PREVIEW %}{% INCLUDE '@phpbb_titania/posting/posting_preview.html' %}{% endif %}
 
-<!-- IF PUBLIC_PREVIEW_MESSAGE -->
+{% if PUBLIC_PREVIEW_MESSAGE %}
 <div class="post bg2" id="preview">
 	<div class="inner">
 	<div class="postbody">
-		<h3>{L_PREVIEW}: {PUBLIC_PREVIEW_SUBJECT}</h3>
+		<h3>{{ lang('PREVIEW') ~ lang('COLON') ~ PUBLIC_PREVIEW_SUBJECT }}</h3>
```
Comment (kasimi) [inline styles/prosilver/template/manage/queue_validate.html]: As @battye mentioned, missing space after colon

File: styles/prosilver/template/posting/posting_buttons.html [hunk 8]
```diff
@@ -37,91 +37,91 @@
...
 		{
-			document.getElementById('bbpalette').value = '{LA_FONT_COLOR}';
+			document.getElementById('bbpalette').value = '{{ lang('FONT_COLOR')|escape('js') }}';
 		}
 	}
 
 // ]]>
 </script>
-<!-- INCLUDEJS {T_ASSETS_PATH}/javascript/editor.js?assets_version={T_ASSETS_VERSION} -->
+{% INCLUDEJS T_ASSETS_PATH ~ '/javascript/editor.js' %}
```
Comment (kasimi) [inline styles/prosilver/template/posting/posting_buttons.html]: Why was the `T_ASSETS_VERSION` removed?

File: styles/prosilver/template/posting/posting_smilies.html [hunk 9]
```diff
@@ -1,22 +1,22 @@
...
+{% INCLUDE 'simple_header.html' %}
 
 <script type="text/javascript">
 // <![CDATA[
 	var form_name = 'postform';
 	var text_name = 'message';
 // ]]>
 </script>
-<!-- INCLUDEJS {T_ASSETS_PATH}/javascript/editor.js?assets_version={T_ASSETS_VERSION} -->
+{% INCLUDEJS T_ASSETS_PATH ~ '/javascript/editor.js' %}
```
Comment (kasimi) [inline styles/prosilver/template/posting/posting_smilies.html]: Why was the `T_ASSETS_VERSION` removed?

File: styles/prosilver/template/posting/posting_topic_review.html [hunk 10]
```diff
@@ -1,68 +1,68 @@
 
 <h3 id="review">
-	<span class="right-box"><a href="#review" onclick="viewableArea(getElementById('topicreview'), true); var rev_text = getElementById('review').getElementsByTagName('a').item(0).firstChild; if (rev_text.data == '{LA_EXPAND_VIEW}'){rev_text.data = '{LA_COLLAPSE_VIEW}'; } else if (rev_text.data == '{LA_COLLAPSE_VIEW}'){rev_text.data = '{LA_EXPAND_VIEW}'};">{L_EXPAND_VIEW}</a></span>
-	{L_TOPIC_REVIEW}: {TOPIC_TITLE}
+	<span class="right-box"><a href="#review" onclick="viewableArea(getElementById('topicreview'), true); var rev_text = getElementById('review').getElementsByTagName('a').item(0).firstChild; if (rev_text.data == '{{ lang('EXPAND_VIEW')|escape('js') }}'){rev_text.data = '{{ lang('COLLAPSE_VIEW')|escape('js') }}'; } else if (rev_text.data == '{{ lang('COLLAPSE_VIEW')|escape('js') }}'){rev_text.data = '{{ lang('EXPAND_VIEW')|escape('js') }}'};">{{ lang('EXPAND_VIEW') }}</a></span>
+	{{ lang('TOPIC_REVIEW') ~ lang('COLON') ~ TOPIC_TITLE }}
```
Comment (kasimi) [inline styles/prosilver/template/posting/posting_topic_review.html]: As @battye mentioned, missing space after colon

File: styles/prosilver/template/plupload.html [hunk 11]
```diff
@@ -64,25 +64,25 @@
...
+{% endif %}
 
 //]]>
 </script>
-<!-- INCLUDEJS {T_ASSETS_PATH}/plupload/plupload.full.min.js?assets_version={T_ASSETS_VERSION} -->
-<!-- INCLUDEJS {T_ASSETS_PATH}/javascript/plupload.js?assets_version={T_ASSETS_VERSION} -->
-<!-- INCLUDEJS {T_TITANIA_ASSETS_PATH}/javascript/plupload.js?assets_version={T_ASSETS_VERSION} -->
+{% INCLUDEJS T_ASSETS_PATH ~ '/plupload/plupload.full.min.js' %}
+{% INCLUDEJS T_ASSETS_PATH ~ '/javascript/plupload.js' %}
+{% INCLUDEJS T_TITANIA_ASSETS_PATH ~ '/javascript/plupload.js' %}
```
Comment (kasimi) [inline styles/prosilver/template/plupload.html]: Why was the `T_ASSETS_VERSION` removed on these 3 lines?

File: styles/prosilver/template/queue_stats_body.html [hunk 12]
```diff
@@ -1,81 +1,81 @@
...
-				<div class="stat-day" title="{dayrow.MONTH}/{dayrow.DAY}">
-					<span style="height: {dayrow.REMAINDER_PCT}%; margin-top: {dayrow.PIXEL_OFFSET}px;"></span>
-					<!-- IF dayrow.NEW_CNT -->
-						<span class="stat-new fade" style="height: {dayrow.NEW_PCT}%" title="{L_REVISION_NEW}: {dayrow.NEW_CNT}">
-						<!-- IF dayrow.NEW_PCT > 7 --><strong>{dayrow.NEW_CNT}</strong><!-- ENDIF -->
+			{% for dayrow in loops.dayrow %}
+				<div class="stat-day" title="{{ dayrow.MONTH }}/{{ dayrow.DAY }}">
+					<span style="height: {{ dayrow.REMAINDER_PCT }}%; margin-top: {{ dayrow.PIXEL_OFFSET }}px;"></span>
+					{% if dayrow.NEW_CNT %}
+						<span class="stat-new fade" style="height: {{ dayrow.NEW_PCT }}%" title="{{ lang('REVISION_NEW') ~ lang('COLON') ~ dayrow.NEW_CNT }}">
```
Comment (kasimi) [inline styles/prosilver/template/queue_stats_body.html]: As @battye mentioned, missing space after colon

File: styles/prosilver/template/queue_stats_body.html [hunk 13]
```diff
@@ -1,81 +1,81 @@
...
+						{% if dayrow.NEW_PCT > 7 %}<strong>{{ dayrow.NEW_CNT }}</strong>{% endif %}
 						</span>
-					<!-- ENDIF -->
-					<!-- IF dayrow.DENIED_CNT -->
-						<span class="stat-denied fade" style="height: {dayrow.DENIED_PCT}%" title="{L_REVISION_DENIED}: {dayrow.DENIED_CNT}">
-						<!-- IF dayrow.DENIED_PCT > 7 --><strong>{dayrow.DENIED_CNT}</strong><!-- ENDIF -->
+					{% endif %}
+					{% if dayrow.DENIED_CNT %}
+						<span class="stat-denied fade" style="height: {{ dayrow.DENIED_PCT }}%" title="{{ lang('REVISION_DENIED') ~ lang('COLON') ~ dayrow.DENIED_CNT }}">
```
Comment (kasimi) [inline styles/prosilver/template/queue_stats_body.html]: As @battye mentioned, missing space after colon

File: styles/prosilver/template/queue_stats_body.html [hunk 14]
```diff
@@ -1,81 +1,81 @@
...
+						{% if dayrow.DENIED_PCT > 7 %}<strong>{{ dayrow.DENIED_CNT }}</strong>{% endif %}
 						</span>
-						<!-- ENDIF -->
-					<!-- IF dayrow.APPROVED_CNT -->
-						<span class="stat-approved fade" style="height: {dayrow.APPROVED_PCT}%" title="{L_REVISION_APPROVED}: {dayrow.APPROVED_CNT}">
-						<!-- IF dayrow.APPROVED_PCT > 7 --><strong>{dayrow.APPROVED_CNT}</strong><!-- ENDIF -->
+						{% endif %}
+					{% if dayrow.APPROVED_CNT %}
+						<span class="stat-approved fade" style="height: {{ dayrow.APPROVED_PCT }}%" title="{{ lang('REVISION_APPROVED') ~ lang('COLON') ~ dayrow.APPROVED_CNT }}">
```
Comment (kasimi) [inline styles/prosilver/template/queue_stats_body.html]: As @battye mentioned, missing space after colon

File: styles/all/template/event/overall_footer_after.html [hunk 15]
```diff
@@ -1,8 +1,8 @@
...
-	<!-- IF .screenshots or SCREENSHOT_UPLOADER -->
-		<!-- INCLUDEJS {T_TITANIA_ASSETS_PATH}/javascript/jquery.magnific-popup.min.js?assets_version={T_ASSETS_VERSION} -->
-	<!-- ENDIF -->
-	<!-- IF S_PLUPLOAD_EXT -->
-		<!-- INCLUDE @phpbb_titania/plupload.html -->
-	<!-- ENDIF -->
-<!-- ENDIF -->
+{% if S_IN_TITANIA %}
+	{% if loops.screenshots|length or SCREENSHOT_UPLOADER %}
+		{% INCLUDEJS T_TITANIA_ASSETS_PATH ~ '/javascript/jquery.magnific-popup.min.js' %}
```
[the remark this answers is not in the document]
Comment (VSEphpbb) [reply styles/all/template/event/overall_footer_after.html]: Thought INCLUDEJS added assets automatically but I guess not in these cases.

**Comment by VSEphpbb** [comment]
All notes should be addressed now

**Comment by VSEphpbb** [comment]
Did email templates too

**Comment by DavidIQ** [comment]
oops...conflicts.

**Comment by VSEphpbb** [comment]
@DavidIQ resolved

**Pull request merged**